### PR TITLE
replacement basic auto

### PR DESCRIPTION
**Pull request by minhlpnguyen**

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -64,5 +67,9 @@ private void configureButtonBindings() {
             
     }
 
+    public Command getAutonomous() {
+        return new Auto(arm, drive, SmartDashboard.getNumber("Auto Drive Time", 5));
```
Comment (jkleiber) [inline src/main/java/frc/robot/RobotContainer.java]: Can we make this not on smartdashboard (just make it a constant)?

**Review by jkleiber — COMMENTED** [review]
Just one comment and then I think we're good. It could be good to also cancel the autonomous command in teleopInit

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (PChild) [reply src/main/java/frc/robot/RobotContainer.java]: B bot should travel 35 feet in 5 seconds at 50% speed, this _probably_ should be a smaller value btw

Comment (jkleiber) [reply src/main/java/frc/robot/RobotContainer.java]: Isn't ChassisSpeeds in m/s? If so then this would be 2.5 meters of movement approximately

Comment (PChild) [reply src/main/java/frc/robot/RobotContainer.java]: ohhh, ok that's fine just very slow

File: src/main/java/frc/robot/commands/auto/Auto.java [hunk 2]
```diff
@@ -17,17 +20,18 @@ public class Auto extends SequentialCommandGroup {
...
+        addRequirements(arm);
+        addRequirements(drive);
 
         addCommands(
-            new InstantCommand(() -> arm.setMode(Mode.SHOOT_HIGH)),
-            new WaitCommand(1),
-            new InstantCommand(() -> arm.setMode(Mode.IDLE)),
-            new ParallelRaceGroup(
+            new ArmMove(arm, Mode.SHOOT_HIGH).raceWith(new WaitCommand(1.0)),
+            new ParallelRaceGroup (
```
Comment (PChild) [inline src/main/java/frc/robot/commands/auto/Auto.java:28]: Is ParallelRaceGroup correct here? Will ArmMove complete and end the race?

Comment (jkleiber) [reply src/main/java/frc/robot/commands/auto/Auto.java:28]: ArmMove never is finished, so it shouldn't win the race against the wait

**Review by jkleiber — APPROVED** [review]
Approved

**Pull request merged**